app/server/BeelieversAuction/auction.server.ts

```diff
@@ -164,7 +166,7 @@ INSERT OR IGNORE INTO stats (key) VALUES ('auction_stats');
 			// TODO: return updated user
 			// and call addDynamicBadges
 
-			return [{ oldRank, newRank }, null];
+			return [{ oldRank, newRank, timestampMs }, null];
```

Comment:
we don't need to return it from here. You can remove this line, and remove `timestampMs` as a return param.
The goal was to use the event timestamp here instead of `new Date` at the beginning of the function

Reply:
Ah, I see! Updating!